### PR TITLE
Replace Jaeger specific exporter with OTLP

### PR DESCRIPTION
**Pull request by skhalymon**
### Summary by OpenAI

Release Notes:

- New Feature: Support for OpenTelemetry traces collector used by Jaeger, Prometheus, etc.
- Bug fix: Replaces Jaeger-specific telemetry exporter with OpenTelemetry Protocol (OTLP) exporter.
- Documentation: Updates environment variables to send logs and traces to OpenTelemetry collector server instead of Jaeger agent.
- Refactor: Replaces Jaeger-specific exporter with OTLP in multiple files.

> "Logs and traces, oh what a sight,
> With OpenTelemetry shining bright.
> Jaeger's gone, but don't you fear,
> Our new exporter is finally here!"

### CODE REVIEW

File: components/restreamer/README.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ You can customize the script behavior by setting the following environment varia
...
 6. `WITH_INITIAL_UPGRADE`: Set to '1' if the system requires a full update before installing (e.g., for Selectel). Default is '0'.
 7. `WITH_FIREWALLD`: Set to '1' if the system requires firewalld instead of ufw (e.g., for Oracle). Default is '0'.
-8. `EPHYR_RESTREAMER_JAEGER_AGENT_IP`: Set the IP address of the Jaeger agent if you want to send traces to Jaeger.
-9. `EPHYR_RESTREAMER_JAEGER_AGENT_PORT`: Set the port of the Jaeger agent if you want to send traces to Jaeger.
-10. `EPHYR_RESTREAMER_JAEGER_SERVICE_NAME`: Set the Jaeger service name for the Ephyr-restreamer traces. Default is the hostname of the machine.
+8. `EPHYR_RESTREAMER_OTLP_COLLECTOR_IP`: Set the IP address of [OpenTelemetry] collector server to send logs to.
+9. `EPHYR_RESTREAMER_OTLP_COLLECTOR_PORT`: Set the port of [OpenTelemetry] collector server to send logs to.
+10. `EPHYR_RESTREAMER_SERVICE_NAME`: Set the service name to collect traces to [OpenTelemetry] collector. Default is the hostname of the machine.
 11. `CLEAR_STATE_ON_RESTART`: Clear `state.json` each restart of Ephyr-restreamer. Default is '0'.
 12. `ALLOWED_IPS`: Set allowed IP addresses to access server. Default is '*'.
```
[the remark this answers is not in the document]
Comment (skhalymon) [reply components/restreamer/README.md:30]: @openai give me example how to validate it on this step

File: common/log/src/lib.rs [hunk 2]
```diff
@@ -52,54 +59,69 @@ impl TelemetryConfig {
...
     }
 
-    /// Set Jaeger service name to collect logs on Jaeger.
+    /// Set current service name to collect traces to [Opentelemetry] collector.
+    ///
+    /// [Opentelemetry]: https://opentelemetry.io
     #[must_use]
-    pub fn jaeger_service_name(mut self, service_name: String) -> Self {
-        self.jaeger_service_name = Some(service_name);
+    pub fn service_name(mut self, service_name: String) -> Self {
```
Comment (PavelKushnirchuk) [inline common/log/src/lib.rs:86]: Why do we need this wrapper ?

Comment (PavelKushnirchuk) [reply common/log/src/lib.rs:86]: I got it. It's for chaining

**Review by PavelKushnirchuk — APPROVED** [review]

File: components/restreamer/src/server.rs [hunk 3]
```diff
@@ -29,8 +29,8 @@ use crate::{
 #[actix_web::main]
 pub async fn run(mut cfg: Opts) -> Result<(), Failure> {
     TelemetryConfig::new(cfg.verbose)
-        .jaeger_endpoint(cfg.jaeger_agent_ip, cfg.jaeger_agent_port)
-        .jaeger_service_name(cfg.jaeger_service_name.clone())
+        .otlp_endpoint(cfg.otlp_collector_ip, cfg.otlp_collector_port)
+        .service_name(cfg.service_name.clone())
         .init();
 
     if cfg.public_host.is_none() {
```
[the remark this answers is not in the document]
Comment (skhalymon) [reply components/restreamer/src/server.rs:36]: It't not correct as it fail with panic

**Pull request merged**